Distribution page (#677)
ros-planning/moveit.ros.org: about/distribution/index.markdown

---
author: admin
comments: false
date: 2022-02-03
layout: page
slug: distribution
title: Distribution
---

<div class='row'>
  <div class='col-sm-12 text-center'>
    <h1>Distributions</h1>
  </div>
  <div class="col-sm-12">
    <h2>What is a Distribution?</h2>

>     <a href="https://wiki.ros.org/Distributions" target="_blank">See wiki.ros.org/Distributions.</a>

Reviewer: ```suggestion
    <a href="https://docs.ros.org/en/rolling/Releases.html" target="_blank">See https://docs.ros.org/en/rolling/Releases.html for a description and a list of available ROS distributions.</a>
```

Reviewer: better use a ROS 2 reference

        </tr>
      </thead>
      <tbody>
        <tr>
          <td><a href="https://moveit.picknik.ai/main/index.html">MoveIt Rolling</a></td>
          <td>2021</td>
          <td><img src="/assets/images/distribution/moveIt-rolling.jpg" alt="MoveIt Rolling" height="100" /></td>
          <td>N/A</td>
        </tr>
        <tr>
          <td><a href="https://moveit.picknik.ai/galactic/index.html">MoveIt 2 Galactic</a></td>
          <td>July 8th, 2021</td>
          <td><img src="/assets/images/distribution/galactic.png" alt="MoveIt 2 Galactic" height="100" /></td>
          <td>November 2022</td>
        </tr>
        <tr>
          <td><a href="https://moveit.picknik.ai/foxy/index.html">MoveIt 2 Foxy Fitzroy</a></td>
          <td>June 5th, 2020</td>
          <td><img src="/assets/images/distribution/moveit2-foxy.png" alt="MoveIt 2 Foxy Fitzroyc" height="100" /></td>
          <td>May 2023</td>
        </tr>
        <tr>
          <td><a href="https://ros-planning.github.io/moveit_tutorials/">MoveIt 1 Noetic</a></td>
          <td>October 13th, 2020</td>
          <td><img src="/assets/images/distribution/MoveIt.png" alt="MoveIt 1 Noetic" height="100" /></td>

>           <td>May 2023</td>

Reviewer: ```suggestion
          <td>May 2025</td>
```

        </tr>
        <tr>
          <td><a href="http://docs.ros.org/en/melodic/api/moveit_tutorials/html/index.html">MoveIt 1 Melodic</a></td>
          <td>June 5th, 2020</td>
          <td><img src="/assets/images/distribution/MoveIt.png" alt="MoveIt 1 Melodic" height="100" /></td>
          <td>May 2023</td>
        </tr>
      </tbody>
    </table>
    <h2>Distribution Details</h2>
    <p>For details on the distributions see each release page. For the supported platforms and versions of common dependencies and other considerations, see the official ROS 2 Target Platforms REP 2000.</p>
    <h2>Future Distributions</h2>
    <p>For details on upcoming features see the roadmap.</p>
    <p>Currently there is a new ROS 2 distribution roughly every year. The following information is a best estimate and is subject to change.</p>
    <p>The expectation is to release new ROS 2 distributions once per year.</p>
    <h2>Rolling Distribution</h2>

>     <p>The Rolling distribution of ROS 2 serves as a staging area for future stable distributions of ROS 2 and as a collection of the most recent development releases. Unlike most stable ROS 2 distributions which have an initial release, a support window during which they are updated, and a definite end of support (see List of Distributions above) the Rolling distribution is continuously updated and is subject to in-place updates which will at times include breaking changes.</p>

Reviewer: This is more or less copied from https://docs.ros.org/en/rolling/Releases.html. afaik, google really doesn't like redundant content (verbatim) and might downrank the page because of this, especially if we reference the ROS 2 distro page directly.

---
author: admin
comments: false
date: 2022-02-03
layout: page
slug: distribution
title: Distribution
---

<div class='row'>
  <div class='col-sm-12 text-center'>
    <h1>Distributions</h1>
    <p class="paragraph-big">
     You're reading the documentation for a version of ROS 2 that has reached its EOL (end-of-life), and is no longer officially supported. If you want up-to-date information, please have a look at Galactic.
    </p>
  </div>
  <div class="col-sm-12">
    <h2>What is a Distribution?</h2>
    <a href="https://wiki.ros.org/Distributions" target="_blank">See wiki.ros.org/Distributions.</a>
    <h2>List of Distributions</h2>
    <table class="table table-striped table--distribution">
      <thead>
        <tr>
          <th>Distro</th>
          <th>Release date</th>
          <th>Logo</th>
          <th>EOL date</th>
        </tr>
      </thead>
      <tbody>
        <tr>

>           <td><a href="https://docs.ros.org/en/crystal/Releases/Release-Foxy-Fitzroy.html">MoveIt Rolling</a></td>

Reviewer: ```suggestion
          <td><a href="https://moveit.picknik.ai/main/index.html</a></td>
```

>           <td>June 5th, 2020</td>

Reviewer: Rolling and Galactic were in 2021

>           <td>June 5th, 2020</td>

Reviewer 2: ```suggestion
          <td>2021</td>
```

          <td></td>
          <td>N/A</td>
        </tr>

        <tr>

>           <td><a href="https://moveit.ros.org/ros/moveit/galactic/2021/07/08/moveit-galactic.html">MoveIt 2 Galactic</a></td>

Reviewer: ```suggestion
          <td><a href="https://moveit.picknik.ai/galactic/index.html</a></td>
```

>           <td>June 5th, 2020</td>

Reviewer 2: ```suggestion
          <td>June, 2021</td>
```

          <td><img src="/assets/images/distribution/galactic.png" height="100" /></td>

>           <td>May 2023</td>

Reviewer: Shouldn't we just use the same EOL dates as the ROS 2 distros? That would be November 2022 for Galactic and May 2023 for Foxy

Reviewer 2: ok I'll update them.

>           <td>May 2023</td>

Reviewer 2: ```suggestion
          <td>November 2022</td>
```

        </tr>

        <tr>
          <td><a href="https://moveit.ros.org/moveit2/ros2/foxy/release/2020/09/04/moveit2-foxy-release.html">MoveIt 2 Foxy Fitzroy</a></td>
          <td>June 5th, 2020</td>
          <td><img src="/assets/images/distribution/moveit2-foxy.png" height="100" /></td>

>           <td>?</td>

Reviewer 2: ```suggestion
          <td>May 2023</td>
```

        </tr>

        <tr>
          <td><a href="https://moveit.ros.org/moveit/ros/noetic/2020/10/13/announcing-moveit-1-1-1-release-for-ros-noetic.html">MoveIt 1 Noetic</a></td>
          <td>October 13th, 2020</td>
          <td><img src="/assets/images/distribution/MoveIt.png" height="100" /></td>
          <td>May 2023</td>
        </tr>

        <tr>

>           <td><a href="https://docs.ros.org/en/crystal/Releases/Release-Foxy-Fitzroy.html">MoveIt 1 Melodic</a></td>

Reviewer: ```suggestion
          <td><a href="http://docs.ros.org/en/melodic/api/moveit_tutorials/html/index.html">MoveIt 1 Melodic</a></td>
```

          <td>June 5th, 2020</td>
          <td><img src="/assets/images/distribution/MoveIt.png" height="100" /></td>
          <td>May 2023</td>
        </tr>

      </tbody>
    </table>
    <h2>Distribution Details</h2>
    <p>For details on the distributions see each release page. For the supported platforms and versions of common dependencies and other considerations, see the official ROS 2 Target Platforms REP 2000.</p>
    <h2>Future Distributions</h2>
    <p>For details on upcoming features see the roadmap.</p>

>     <p>Currently there is a new ROS 2 distribution roughly every 6 months. The following information is a best estimate and is subject to change.</p>

Reviewer: ```suggestion
    <p>Currently there is a new ROS 2 distribution roughly every year. The following information is a best estimate and is subject to change.</p>
```
